iopipe/report.py

```diff
@@ -61,6 +60,7 @@ def __init__(self, config, context):
             'installMethod': self.config.get('install_method'),
             'plugins': self.plugins,
             'processId': constants.PROCESS_ID,
+            'timestamp': int(time.time() * 1000),
```

Comment:
This is a really great fix if I'm reading this right -- the agent was previously reporting its end time as `timestamp`? (which we use like start time irrc?)

Reply:
Yes, this should put it in sync with the node.js implementation.

Reply:
Awesome

Reply:
Instead of a new timestamp here, since we have `self.start_time`, we should use that